datasets/nwm-archive.yaml

```diff
@@ -67,6 +81,13 @@ DataAtWork:
       URL: https://nbviewer.jupyter.org/gist/rsignell-usgs/d3dfaf3cd3d8b39894a69b22127dfe38
       AuthorName: Rich Signell
       AuthorURL: https://www.usgs.gov/staff-profiles/richard-p-signell
+    - Title: Explore Repository of Tutorials on National Water Model V2.1 Retrospective Dataset in Zarr
+      URL: https://github.com/NCAR/rechunk_retro_nwm_v21/tree/main/notebooks
+      AuthorName: James McCreight
+      AuthorURL: https://github.com/jmccreight
+  Zarr v2.1 Conversion Code Repository:

```

Comment:
@Patrick-Keown sorry, didn't see this earlier. This block is problematic. It needs to be under either a `Publications`, `Tools & Applications`, or `Tutorials` heading like the others. And it'll need to have a URL as well. Please also be careful of the hypens, there should be just one for the whole item (like the ones above).

Reply:
Thanks Joe. Issued resolved.